pkg/util/templates/templates.go

```diff
@@ -70,6 +76,29 @@ type multiWindowMultiBurnAlertRule struct {
 	burnRate    string
 }
 
+
+func alertThreshold(windowSize, percent, label, burnRate string) string {
+
+	rule := "1-(sum(sum_over_time(probe_success{" + label + "}[" + windowSize + "]))" +
+		"/ sum(count_over_time(probe_success{" + label + "}[" + windowSize + "])))" +
+		"> (" + burnRate + "*(1-" + percent + "))"
+
+	return rule
+}
+
+func sufficientProbes(windowSize, label string) string {
+	window, _ := prometheus.ParseDuration(windowSize)
+	window_duration := time.Duration(window)
+	mPeriod, _ := prometheus.ParseDuration(serviceMonitorPeriod)
+	mPeriod_duration := time.Duration(mPeriod) 
+	necessaryProbesInWindow := int(window_duration.Minutes()/mPeriod_duration.Minutes() * 0.5)
+
+	rule := "sum(count_over_time(probe_success{" + label + "}[" + windowSize + "]))" +
+		" > " + strconv.Itoa(necessaryProbesInWindow)
+
+	return rule
+}
+
 //render creates a monitoring rule for the defined multiwindow multi-burn rate alert
 // Sample result as yaml
 /*
```

Comment:
this comment seems super outdated now

Reply:
yes I would also just drop these kind of comments